crates/holochain_types/src/web_app/web_app_bundle.rs

```diff
@@ -12,12 +10,12 @@ pub struct WebAppBundle(Bundle<WebAppManifest>);
 
 impl WebAppBundle {
     /// Construct from raw bytes
-    pub fn decode(bytes: &[u8]) -> MrBundleResult<Self> {
+    pub fn decode(bytes: bytes::Bytes) -> MrBundleResult<Self> {
         Bundle::decode(bytes).map(WebAppBundle)
     }
 
     /// Returns the bytes of the zip file containing the Web UI contained inside this WebAppBundle
-    pub async fn web_ui_zip_bytes(&self) -> MrBundleResult<Cow<'_, ResourceBytes>> {
```

Comment:
Cows are for farms.

(but actually I don't understand the utility -- seems like we would rather be explicit when we're cloning things).